src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -51,25 +51,11 @@ class OverseasEntitySubmissionDtoValidatorTest {
     @Mock
     private PresenterDtoValidator presenterDtoValidator;
     @Mock
-    private OverseasEntityDueDiligenceValidator overseasEntityDueDiligenceValidator;
-    @Mock
-    private BeneficialOwnersStatementValidator beneficialOwnersStatementValidator;
-    @Mock
     private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
     @Mock
-    private BeneficialOwnerIndividualValidator beneficialOwnerIndividualValidator;
-    @Mock
-    private BeneficialOwnerCorporateValidator beneficialOwnerCorporateValidator;
-    @Mock
-    private DueDiligenceValidator dueDiligenceValidator;
-    @Mock
-    private BeneficialOwnerGovernmentOrPublicAuthorityValidator beneficialOwnerGovernmentOrPublicAuthorityValidator;
-    @Mock
     private DueDiligenceDataBlockValidator dueDiligenceDataBlockValidator;
     @Mock
-    private ManagingOfficerIndividualValidator managingOfficerIndividualValidator;
-    @Mock
-    private ManagingOfficerCorporateValidator managingOfficerCorporateValidator;
+    private OwnersAndOfficersDataBlockValidator ownersAndOfficersDataBlockValidator;
 
     private EntityDto entityDto = EntityMock.getEntityDto();
```

Comment:
intellij has highlighted most of the private dto class level variables and recommends making them 'final'.

Reply:
Added final same for below